app/Makefile

```diff
@@ -68,7 +68,11 @@ ifneq ($(TARGET_NAME),TARGET_NANOS)
     DEFINES += COMPILE_TEXTUAL
 endif
 
-APP_STACK_MIN_SIZE := 1600
+ifeq ($(TARGET_NAME),TARGET_NANOS)
+	APP_STACK_MIN_SIZE := 1600
+else
+	APP_STACK_MIN_SIZE := 3800
```

Comment:
In fact should be >1736.
There are 2K that are not really here (but currently exposed over Speculos)

Reply:
or 1752 if you want to keep your current margin